src/wizard/java/spring.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Spring
+doc_link: https://https://docs.sentry.io/platforms/java/guides/spring/
+support_level: production
+type: framework
+---
+
+<Alert level="info">
+    Sentry's integration with Spring supports Spring Framework 5.1.2 and above to report unhandled exceptions and optional user information. If you're on an older version, use <a href=https://docs.sentry.io/platforms/java/guides/spring/legacy/>our legacy integration</a>.
+</Alert>
+
+Install Sentry's integration with Spring via Maven, Gradle, or SBT:
+
+```xml {tabTitle:Maven}
+<dependency>
+    <groupId>io.sentry</groupId>
+    <artifactId>sentry-spring</artifactId>
+    <version>{{ packages.version('sentry.java', '3.2.0') }}</version>
+</dependency>
+```
+
+```groovy {tabTitle:Gradle}
+implementation 'io.sentry:sentry-spring:{{ packages.version('sentry.java', '3.2.0') }}'
+```
+
+```scala {tabTitle: SBT}
+libraryDependencies += "io.sentry" % "sentry-spring" % "{{ packages.version('sentry.java', '3.2.0') }}"
+```
+
+For other dependency managers see the [central Maven repository](https://search.maven.org/artifact/io.sentry/sentry-spring).
+
+Configure Sentry as soon as possible in your application's lifecycle:
+
+<Note>
+
+The `sentry-spring` library provides `@EnableSentry` annotation that registers all required Spring beans. `@EnableSentry` can be placed on any class annotated with [@Configuration](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/annotation/Configuration.html) including the main entry class in Spring Boot applications annotated with [@SpringBootApplication](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/SpringBootApplication.html).
+
+</Note>
+
+```java {tabTitle:Java}
```

Comment:
Without some heading it looks odd both snippets one on top of the other
```suggestion
### Java
```java {tabTitle:Java}
```

Reply:
agreed. I'll fix.